src/js/view/template/week/time.hbs

```diff
@@ -53,7 +53,7 @@
                 {{/if}};">{{{comingDuration-tmpl model}}}</div>
             {{/if}}
             </div>
-            {{#unless croppedEnd}}<div class="{{CSS_PREFIX}}time-resize-handle handle-x" style="margin-left: {{@root.styles.paddingLeft}};">&nbsp;</div>{{/unless}}
+            {{#unless croppedEnd}}<div class="{{CSS_PREFIX}}time-resize-handle handle-x{{#if model.isReadOnly}} {{CSS_PREFIX}}hide{{/if}}">&nbsp;</div>{{/unless}}
```

Comment:
I think you don't have to add another class. You can just remove the element. Please refer to month view.
```html
{{#unless (or @root.isReadOnly model.isReadOnly)}}<span class="{{CSS_PREFIX}}weekday-resize-handle handle-y" style="line-height:{{@root.scheduleHeight}}px;">&nbsp;</span>{{/unless}}
```